src/sidebar/services/test/root-thread-test.js

```diff
@@ -288,7 +290,7 @@ describe('rootThread', function() {
 
     it('does not filter annotations when not in the sidebar', function() {
       fakeBuildThread.reset();
-      fakeStore.state.base.isSidebar = false;
```

Comment:
Can you walk me through this `base` naming here?

Reply:
Good Q. Yes, the "base" is the temporary namespace for all non-namespaced modules.

The first step of this process was to add multiple reducers, but because of that, we needed to give a superficial namespace to the existing store because they need a name for `combineReducers` to work. I called this "base". Once we convert all the modules, "base" will go away so that's why you see it deleted here. 

Once my next PR goes up, there should be no more "base"